rhc-ose-ansible/roles/disconnected-git/defaults/main.yaml

```diff
@@ -0,0 +1,15 @@
+---
+git_repo_home: "/opt/git"
+ose_git_repo_home: "{{ git_repo_home }}/openshift"
+git_user: "git"
+
+# List of Authorized Keys to Add
+git_user_authorized_keys:
+
+ose_example_repos:
```

Comment:
I suppose since this is defaults (at first I thought this was under tasks)
it should stay here, but I would like to include a sample inventory that
shows overriding the default values, just so it's documented.

On Tue, May 24, 2016, 9:44 PM Andrew Block <notifications@github.com> wrote:

> In rhc-ose-ansible/roles/disconnected-git/defaults/main.yaml
> <https://github.com/rhtconsulting/rhc-ose/pull/171#discussion_r64503825>:
>
> > @@ -0,0 +1,15 @@
> > +---
> > +git_repo_home: "/opt/git"
> > +ose_git_repo_home: "{{ git_repo_home }}/openshift"
> > +git_user: "git"
> > +
> > +# List of Authorized Keys to Add
> > +git_user_authorized_keys:
> > +
> > +ose_example_repos:
>
> @etsauer <https://github.com/etsauer> the entire contents of this file or
> a particular variable?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/rhtconsulting/rhc-ose/pull/171/files/7946bdbd8b2baf70fb2e507a94faa765a73c157b#r64503825>
>